lib/parseOpenDataXml.js

```diff
@@ -1,4 +1,4 @@
-var parser = require("xml2json");
+var parser = require("p3x-xml2json");
```

Comment:
do we want add an TODO comment to change again after resolving this issue https://github.com/buglabs/node-xml2json/issues/177

this p3x package looks like a tmp-fix package.